File: app/src/main/java/org/michaelbel/moviemade/rest/model/Person.java
package org.michaelbel.moviemade.rest.model;

import com.google.gson.annotations.SerializedName;

import org.michaelbel.tmdb.TmdbObject;

import java.io.Serializable;
import java.util.List;

import io.realm.annotations.Ignore;

public class Person extends TmdbObject implements Serializable {

    @SerializedName("id")
    public int id;

    @SerializedName("name")
    public String name;

    @SerializedName("gender")
    public int gender;

    @SerializedName("birthday")
    public String birthday;

    @SerializedName("deathday")
    public String deathday;

    @SerializedName("place_of_birth")
    public String birthPlace;

    @SerializedName("profile_path")
    public String profilePath;

    @SerializedName("biography")
    public String bio;

    @SerializedName("adult")
    public boolean adult;

    @SerializedName("popularity")
    public double popularity;

    @SerializedName("imdb_id")
    public String imdbId;

    @SerializedName("homepage")
    public String homepage;

    @Ignore
    @SerializedName("also_known_as")
    public List<String> names;
}

File: app/src/main/java/org/michaelbel/moviemade/data/dao/Person.kt
package org.michaelbel.moviemade.data.dao

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Person(
    @SerializedName("birthday") val birthDay: String,
    @SerializedName("known_for_department") val knownForDepartment: String,
    @SerializedName("deathday") val deathDay: String,
    @SerializedName("id") val id: Int,
    @SerializedName("name") val name: String,
    @SerializedName("also_known_as") val also_known_as: List<String>,
    @SerializedName("gender") val gender: Int,
    @SerializedName("biography") val biography: String,
    @SerializedName("popularity") val popularity: Float,
    @SerializedName("place_of_birth") val placeOfBirth: String,
    @SerializedName("profile_path") val profilePath: String,
    @SerializedName("adult") val adult: Boolean,
    @SerializedName("imdb_id") val imdb_id: String,
    @SerializedName("homepage") val homepage: String,

    @SerializedName("known_for") val knownFor: List<Movie>
) : Serializable